GoogleSignIn/Sources/GIDVerify/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
I believe it is more clear to make `GoogleSignIn/Sources/GIDVerify/...` this instead: `GoogleSignIn/Sources/GIDVerifyAccountDetail/...`.

Reply:
Done.

Reply:
Resolving as addressed in https://github.com/google/GoogleSignIn-iOS/pull/379/commits/990f05143febc13f79cf62016711dd77f4d6c046. Please reopen if needed.